core/src/test/java/com/google/zxing/aztec/encoder/EncoderTest.java

```diff
@@ -577,15 +577,21 @@ private static void testHighLevelEncodeString(String s, String expectedBits) {
     BitArray bits = new HighLevelEncoder(s.getBytes(StandardCharsets.ISO_8859_1)).encode();
     String receivedBits = stripSpace(bits.toString());
     assertEquals("highLevelEncode() failed for input string: " + s, stripSpace(expectedBits), receivedBits);
-    assertEquals(s, Decoder.highLevelDecode(toBooleanArray(bits)));
+    try {
+      assertEquals(s, Decoder.highLevelDecode(toBooleanArray(bits)));
+    } catch (FormatException e) {
```

Comment:
Not sure if there's another way that you'd prefer this to be handled @srowen.

Reply:
Just let the method throw FormatException. We want it to fail the test if unexpected.